### PR TITLE
7903868: Release jtreg 7.5

### PR DESCRIPTION
**Pull request by sormuras**
Please review this update to the `CHANGELOG.md` file. This change also includes the increase of the version number for a future release.

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [CODETOOLS-7903868](https://bugs.openjdk.org/browse/CODETOOLS-7903868): Release jtreg 7.5 (**Task** - P3)

### Reviewers
 * [Vicente Romero](https://openjdk.org/census#vromero) (@vicente-romero-oracle - no project role)
 * [Jan Lahoda](https://openjdk.org/census#jlahoda) (@lahodaj - **Reviewer**)
 * [Jaikiran Pai](https://openjdk.org/census#jpai) (@jaikiran - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jtreg.git pull/231/head:pull/231` \
`$ git checkout pull/231`

Update a local copy of the PR: \
`$ git checkout pull/231` \
`$ git pull https://git.openjdk.org/jtreg.git pull/231/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 231`

View PR using the GUI difftool: \
`$ git pr show -t 231`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jtreg/pull/231.diff">https://git.openjdk.org/jtreg/pull/231.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jtreg/pull/231#issuecomment-2419157612)

### CODE REVIEW

**Review by vicente-romero-oracle — APPROVED** [review]
looks good

File: make/build-support/version-numbers [hunk 1]
```diff
@@ -23,7 +23,7 @@
 # questions.
 #
 
-JTREG_VERSION=7.5
+JTREG_VERSION=7.6
```
Comment (jaikiran) [inline make/build-support/version-numbers:26]: Hello Christian, is it intentional that this version says `7.6` and the "unreleased" link in the changelog for the next release uses `7.5+1`?

Comment (sormuras) [reply make/build-support/version-numbers:26]: Yes. `7.6` here indicates the new version for the upcoming release(s) and `7.5+1` in the changelog refers to git tags for detailed comparisons.

**Review by jaikiran — APPROVED** [review]

File: make/build-support/version-numbers (hunk 1 above)
Comment (jaikiran) [reply make/build-support/version-numbers:26]: The `+1` vs `.1` has always confused me. When I read that diff I thought it represented 7.5.1.

**Pull request closed without merging**